scripts/patches/group.diff.mustache

```diff
@@ -0,0 +1,13 @@
+diff --git a/cluster-scope/base/user.openshift.io/groups/{{name}}/group.yaml b/cluster-scope/base/user.openshift.io/groups/{{name}}/group.yaml
+new file mode 100644
+--- /dev/null
++++ b/cluster-scope/base/user.openshift.io/groups/{{name}}/group.yaml
+@@ -0,0 +1,{{users.length}}@@
```

Comment:
@tumido this is where it breaks already, at least with the golang implementation of mustache. There's no `.length` implemented.
Removing the `@@` line results in an empty applied patch, so the number has to be accurate.

I'm investigating some of the jsonnet/ksonnet tooling mentioned at https://jsonnet.org/articles/kubernetes.html